### PR TITLE
GET /api/v2/schema/<schema_id>

### PR DESCRIPTION
**Pull request by JR40159**

### CODE REVIEW

File: backend/src/routes/v2/schema/getSchemas.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ export const getSchemas = [
   async (req: Request, res: Response<GetSchemaResponse>) => {
     const { query } = parse(req, getSchemasSchema)
 
-    const schemas = await findSchemasByKind(query.kind)
+    const schemas = await getSchemasByKind(query.kind)
```
Comment (a3957273) [inline backend/src/routes/v2/schema/getSchemas.ts:27]: Am tempted to say that request functions should have names like `getSchemas` and `postResult`, etc.  Services should be independent of HTTP request language and so use phrasing like `createSchema` or `findSchemas` or `readSchema`.

File: backend/test/services/schema.spec.ts [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
     find: vi.fn(() => ({ sort: vi.fn(() => ['schema-1', 'schema-2']) })),
+    findOne: vi.fn(),
   }
 
   const Schema: any = vi.fn(() => ({
     save: mockedMethods.save,
   }))
   Schema.find = mockedMethods.find
-  Schema.deleteOne = mockedMethods.deleteOne
+  ;(Schema.findOne = mockedMethods.findOne), (Schema.deleteOne = mockedMethods.deleteOne)
```
Comment (a3957273) [inline backend/test/services/schema.spec.ts:18]: What is going on here?!  Put it on multiple lines!

**Review by a3957273 — CHANGES_REQUESTED** [review]

File: backend/src/routes/v2/schema/getSchemas.ts (hunk 1 above)
Comment (JR40159) [reply backend/src/routes/v2/schema/getSchemas.ts:27]: I made this change to follow the convention you've implemented in your PR....

File: backend/test/services/schema.spec.ts (hunk 2 above)
Comment (JR40159) [reply backend/test/services/schema.spec.ts:18]: Caused by running npm run style, not sure why

**Review by a3957273 — APPROVED** [review]

**Pull request merged**